### PR TITLE
[FIX] Links and upload paths when running in a subdir

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
Maybe related to https://github.com/RocketChat/Rocket.Chat/issues/13849

### CODE REVIEW

**Comment by ura14h** [comment]
@MarcosSpessatto 
The commit #13746 may give your pull-request a impact. I recommend to merge the latest master and check your patch.

**Comment by MarcosSpessatto** [comment]
@ura14h thanks for let me know, but this branch is updated with the latest `develop` changes, what means that your changes already is there. :slightly_smiling_face:

File: app/theme/server/server.js [hunk 1]
```diff
@@ -80,7 +81,8 @@ export const theme = new class {
 
 			currentHash = crypto.createHash('sha1').update(data.css).digest('hex');
 			currentSize = data.css.length;
-			Inject.rawHead('css-theme', `<link rel="stylesheet" type="text/css" href="/theme.css?${ currentHash }">`);
+			const themePath = `/theme.css?${ currentHash }`;
+			Inject.rawHead('css-theme', `<link rel="stylesheet" type="text/css" href=${ getURL(themePath) }>`);
```
Comment (geekgonecrazy) [inline app/theme/server/server.js]: doesn't this still need to be in `"` ?

Comment (MarcosSpessatto) [reply app/theme/server/server.js]: Does not need, does not change behavior, but I'll add to keep the pattern.

**Pull request merged**